Tests/SentryTests/Networking/SentryReachabilityTests.m

```diff
@@ -47,15 +64,17 @@ - (void)testConnectivityRepresentations
 - (void)testMultipleReachabilityObservers
 {
     SentryReachability *reachability = [[SentryReachability alloc] init];
+    // Disable the rechability callbacks, cause we call the callbacks manually.
+    // Otherwise, the rechability callbacks kick it at some point and make the tests flaky.
```

Comment:
```suggestion
    // Disable the reachability callbacks, cause we call the callbacks manually.
    // Otherwise, the reachability callbacks are called during later unrelated tests causing flakes.
```

Reply:
Thanks, just enabled spelling and grammar again in Xcode 😄